src/opera/commands/deploy.py

```diff
@@ -45,8 +57,23 @@ def deploy(args):
         print("{0} is not a positive number!".format(args.workers))
         return 1
 
-    # TODO(@tadeboro): This should be part of the init command that we do not
-    # have yet.
+    if storage.exists("instances"):
+        if args.resume:
+            print("The resume deploy option might have unexpected consequences on the already deployed blueprint.")
+            question = prompt_yes_no_question()
+            if not question:
+                return 0
+        elif args.force:
+            print("The force deploy option might have unexpected consequences on the already deployed blueprint.")
+            question = prompt_yes_no_question()
+            if question:
+                storage.remove("instances")
+            else:
+                return 0
+        else:
+            print("The instance model already exists. To redeploy use --force option or --resume to continue.")
```

Comment:
Yes, good call I will expand the message a little bit. The third option you mentioned can be skipped for now since `redeploy` may be the new command that will need to be implemented.

Reply:
The new message is: "The instance model already exists. Use --resume to continue or --force to ignore current deployment state and start over the deployment."